js/pillbox.js

```diff
@@ -109,7 +117,7 @@
 
 			if (!$target.hasClass('pill')) {
 				$item = $target.parent();
-				if (this.$element.attr('data-readonly') === undefined) {
+				if (typeof this.$element.attr('data-readonly') === 'undefined') {
```

Comment:
typeof undefined is not necessary. if our es5 lint setting are yelling we need to adjust them 
https://github.com/jamesallardice/jslint-error-explanations/issues/10#issuecomment-18273885